<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nucleation</class>
 <widget class="QDialog" name="nucleation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="outputScreen">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>301</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listScreen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>281</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>RUN</string>
   </property>
  </widget>
  <widget class="QwtPlot" name="maximosQwtPlot">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>321</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="SumasQwtPlot">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>330</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="lineEdit1Button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lineEdit2Button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="defaultCheckBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>97</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="cellSizeSpinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
